.gitpod.Dockerfile

```diff
@@ -1,5 +1,18 @@
-FROM deniswsrosa/couchbase7.0.beta-gitpod
+FROM couchbase:latest
+
+RUN echo "* soft nproc 20000\n"\
+"* hard nproc 20000\n"\
+"* soft nofile 200000\n"\
+"* hard nofile 200000\n" >> /etc/security/limits.conf
 
 #Simple example on how to extend the image to install Java and maven
```

Comment:
```suggestion
#Simple example on how to extend the image to install Node.js and NPM
```

Reply:
Thanks @nithishr for catching this typo. Removed this comment line to re-use it.